tests/foreman/api/test_capsulecontent.py

```diff
@@ -1312,8 +1313,9 @@ def test_positive_remove_capsule_orphans(
             data={'environment_id': function_lce_library.id}
         )
         result = capsule_configured.nailgun_capsule.content_lifecycle_environments()
-        assert len(result['results']) == 1
-        assert result['results'][0]['id'] == function_lce_library.id
+        if not pytestconfig.option.n_minus:
```

Comment:
@jyejare, do I guess correctly the need for this `if` is the fact that n_minus capsule is session-scoped like and thus contains more LCEs?

This test case is supposed to use the function-scoped `capsule_configured` (not the module scoped one) for the reason of verification that all RPM artifacts are cleared after orphan cleanup. This will not happen for a session-scoped capsule, since any previously run test (which syncs at least one yum repo) will leave RPM artifacts on the capsule and the test will fail on L1351.

In other words, I don't think this test case is good for n_minus scenario.

Reply:
Hello @vsedmik for comment, Yes the capsule here is a single capsule and given by CI where Satellite is at higher version and capsule at lower (We call this as n-minus).

Since the capsule has all the data from all the tests (like LCEs in this case), we had to skip some assertions like these. Apart from these assertions rest assertions are being validated. But As per the design of n-minus setup we cant have on-demand capsules like the scoped caspules once etc.

Since this is a smart test collection (that is without markers) , to uncollect some test we have to introduce some uncollect marker OR keep the test with custom assertions. I would vote for second.

Reply:
@vsedmik Atleast, I dont see the test failing after I removed these 2 assertions, but If you feel like we should be just skipping it, I can do it!

Reply:
@jyejare Hmm, does it pass when run standalone or within the whole test suite? Do you know the test order run before this one?

Reply:
@vsedmik AFAIR, it passes and passes within the whole suite. I can PM you the order because the list would be big here to paste.